### DISCUSSION TITLE
Repository Unavailable

### DISCUSSION THREAD

**Discussion by tanishkaa19** [community/community#139351] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have forked a repository from my friend but it is showing temporary unavailable. How can I resolve this issue ?

**Comment by alx-mp4** [comment]
Hi @tanishkaa19, would you be okay with posting screenshots or links to the repository in question?

**Reply by tanishkaa19** [reply]
<img width="1452" alt="Screenshot 2024-09-23 at 3 36 45 PM" src="https://github.com/user-attachments/assets/fc7a82fd-9bb3-452c-b622-af07e7d6fe67">

**Reply by alx-mp4** [reply, marked as answer]
Hmm yeah, I am not sure if we could do much in this case. If the issue has persisted for longer than expected, I would advise opening a ticket for the GitHub Support team to look into it. You can actually see there it says the same. Hopefully this helps!

**Comment by diya-el-fadhil** [comment]
Try cloning the repo locally

here is the code : git clone https://github.com/username/repo-name.git

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @tanishkaa19 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Repositories category. I've gone ahead and moved it for you. Good luck!
